jobs/gradle.yml

```diff
@@ -0,0 +1,50 @@
+# gradle
+#
+# Job responsible for building Gradle Wrapper based projects.
+#
+
+parameters:
+  jdkVersions:
```

Comment:
If possible, I would add all versions starting from 8 till 13 and make is up to user of this template which once to actually use.  Is this possible?

Reply:
This list are the default versions and if the user wants to test on others as well it can be done in the azure-pipelines.yml file

Reply:
so additional versions will be injected and picked up by this config too?

Reply:
Nope, if you call this job with the `jdkVersions` parameter you will basically overwrite the values.